ale/drivers/kaguya_drivers.py

```diff
@@ -88,6 +114,46 @@ def _tc_id(self):
         """
         return spice.bods2c("LISM_{}".format(super().instrument_id))
 
+    @property
+    def spacecraft_name(self):
+        """
+        Returns "MISSION_NAME" as a proxy for spacecraft_name.
+
+        No NAIF code exists for the spacecraft name 'SELENE-M.'  The NAIF code
+        exists only for 'SELENE' or 'KAGUYA' -- 'SELENE' is captured as
+        'MISSION_NAME'
+        
+        Returns
+        -------
+        : str
+          mission name
+        """
+        return self.label.get('MISSION_NAME')
+
+    @property
+    def spacecraft_clock_start_count(self):
+        """
+        Returns the start clock count string from the PDS3 label.
+
+        Returns
+        -------
+        : float
+          spacecraft clock stop count in seconds
+        """
+        return str(self.label['SPACECRAFT_CLOCK_START_COUNT'].value)
+    
+    @property
+    def spacecraft_clock_stop_count(self):
+        """
+        Returns the stop clock count string from the PDS3 label.
+
+        Returns
+        -------
+        : float
+          spacecraft clock stop count in seconds
+        """
+        return str(self.label['SPACECRAFT_CLOCK_STOP_COUNT'].value)
```

Comment:
The start/stop times need to get the corrected value or else you'll have inaccuracies. clock_stop_time and clock_start_time should be removed if they are not being used in ISD generation.